### PR TITLE
add surgery fail chance for not using painkillers

### PR DESCRIPTION
**Pull request by SomeguyManperson**
#  General Documentation

### Intent of your Pull Request

Adds a failure chance to surgeries unless the patient is under either a painkiller or anesthetic gas. If you REALLY need to remove it you can also get them super drunk as a ghetto alternative.

some chems, currently only miner's salve and the drink painkiller will provide reductions to the penalty

### Why is this change good for the game?
gives an actual use to morphine and anesthetics aside from being an antagonist
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.
Surgeries now have an innate 0.5 modifier attached to the surface/chem multiplier, this can can be slightly reduced with certain chemicals and fully removed by using a knockout gas, stasis bed, morphine, or in a pinch a lot of alcohol
### What should players be aware of when it comes to the changes your PR is implementing?
need to use some amount of painkillers to do surgery relatively successfully
also stasis bed has had its surgery success multiplier reduced from 100% to 80%, which can be fixed with sterilizer spray since it works as an alternative to painkillers
### What general grouping does this PR fall under? 
Surgery

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
n
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
currently only the painkiller drink and miner's salve give a 0.5* modifier to the failure chance, the failure modifier itself is a 50% reduction in success rates for surgery

# Changelog

:cl:  
tweak: surgeries now have a drastically lower success rate when the target is not under anesthetics or painkillers
tweak: stasis beds now only have 80% surgery success rate but can be used without anesthetics or painkillers 
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
pog

**Comment by alexkar598** [comment]
This is a bad change, can it offer some oppotunities for RP? Maybe? Not really, the only thing this gives is the ability for traitor doctors to do brainwashing and shit except that already happens successfully without anesthetics.

Now why this isn't just not fixing anything but actually counter intuitive:
Talking with your doctor is actually a big part of the tiny bit of RP that happens in medical. And the other point which is the most important here: Noone likes to stare at a black screen for 10 minutes while the incompetant doctor fumbles around trying to fix you up while youre unconscious. 

And finally the silly bits that make this PR a fucking meme:

- Encourages the use of morphine, an addictive chem that will fuck over your entire round as you develop an addiction.
- Stasis bed change barely documented and feels like its being passed under the radar.
- There's a fucking feature freeze
- You added yet another variable to /mob/living, which is already bloated with dumb variables.


This is a repeat of nickvr's xenobio changes. mmmm yes X is bad, Y is good, good change without anyone stopping to take the time to think why they were doing what they were doing. In xeno's case it was xeno is bad, making their job require others good which was completly silly as it made them useless and here its ??? bad, anesthetics good, much rp.

**Comment by SomeguyManperson** [comment]
Ok so we should remove anesthetics for having no effective gameplay reason to exist thanks for your opinion

**Comment by alexkar598** [comment]
Morphine needs to stay because of antag play. Anesthetics on the other hand... doesnt really have any use and could be removed, or repurposed as a way to make field surgery work.

**Comment by Marmio64** [comment]
could you add Inocybe Shine as a drink that helps reduce failure chance? Mostly for ashwalkers doing ghetto surgery.

**Comment by SomeguyManperson** [comment]
Also
Morphine's addiciton threshold is at 25 units so you wont get overdosed unless someone is like dumping a sleeper into you or forcefeeding you an entire bottle of the shit and by that logic any medicine being applied is bad because someone COULD overdose you and put you in hell until its purged

The stasis bed change is documented in the changelog and the wiki section since it has to have some downside for acting in the same place as things that tend to knock you out

"Not being allowed to do anything for 10 minutes through no fault of your own" already occurs plenty in departments like security

And while it's fun talking to someone while sawing their ribcage open which this will limit somewhat but not outright remove it gets old hearing people say "ow" every time you do a circular saw step

**Review by JamieD1 — APPROVED** [review]

**Comment by alao3alao3** [comment]
so how are we going to tell the public about this change, or are they just going to have to find out by themselves cause not even the Wiki nor Jamie knew what this speedmerge entailed.

**Comment by Moggram** [comment]
This is fucking pain full.
